components/form-token-field/test/index.js

```diff
@@ -126,147 +124,147 @@ maybeDescribe( 'FormTokenField', function() {
 			wrapper.setState( {
 				tokens: fixtures.specialTokens.textUnescaped,
 			} );
-			expect( getTokensHTML() ).to.deep.equal( fixtures.specialTokens.htmlUnescaped );
+			expect( getTokensHTML() ).toEqual( fixtures.specialTokens.htmlUnescaped );
 		} );
 	} );
 
 	describe( 'suggestions', function() {
 		it( 'should not render suggestions unless we type at least two characters', function() {
-			expect( getSuggestionsText() ).to.eql( [] );
+			expect( getSuggestionsText() ).toEqual( [] );
 			setText( 'th' );
-			expect( getSuggestionsText() ).to.eql( fixtures.matchingSuggestions.th );
+			expect( getSuggestionsText() ).toEqual( fixtures.matchingSuggestions.th );
 		} );
 
 		it( 'should remove already added tags from suggestions', function() {
 			wrapper.setState( {
 				tokens: Object.freeze( [ 'of', 'and' ] ),
 			} );
-			expect( getSuggestionsText() ).to.not.include.members( getTokensHTML() );
+			expect( getSuggestionsText() ).not.toEqual( getTokensHTML() );
 		} );
 
 		it( 'suggestions that begin with match are boosted', function() {
 			setText( 'so' );
-			expect( getSuggestionsText() ).to.deep.equal( fixtures.matchingSuggestions.so );
+			expect( getSuggestionsText() ).toEqual( fixtures.matchingSuggestions.so );
 		} );
 
 		it( 'should match against the unescaped values of suggestions with special characters', function() {
 			setText( '& S' );
 			wrapper.setState( {
 				tokenSuggestions: fixtures.specialSuggestions.textUnescaped,
 			} );
-			expect( getSuggestionsText() ).to.deep.equal( fixtures.specialSuggestions.matchAmpersandUnescaped );
+			expect( getSuggestionsText() ).toEqual( fixtures.specialSuggestions.matchAmpersandUnescaped );
 		} );
 
 		it( 'should match against the unescaped values of suggestions with special characters (including spaces)', function() {
 			setText( 's &' );
 			wrapper.setState( {
 				tokenSuggestions: fixtures.specialSuggestions.textUnescaped,
 			} );
-			expect( getSuggestionsText() ).to.deep.equal( fixtures.specialSuggestions.matchAmpersandSequence );
+			expect( getSuggestionsText() ).toEqual( fixtures.specialSuggestions.matchAmpersandSequence );
 		} );
 
 		it( 'should not match against the escaped values of suggestions with special characters', function() {
 			setText( 'amp' );
 			wrapper.setState( {
 				tokenSuggestions: fixtures.specialSuggestions.textUnescaped,
 			} );
-			expect( getSuggestionsText() ).to.deep.equal( fixtures.specialSuggestions.matchAmpersandEscaped );
+			expect( getSuggestionsText() ).toEqual( fixtures.specialSuggestions.matchAmpersandEscaped );
 		} );
 
 		it( 'should match suggestions even with trailing spaces', function() {
 			setText( '  at  ' );
-			expect( getSuggestionsText() ).to.deep.equal( fixtures.matchingSuggestions.at );
+			expect( getSuggestionsText() ).toEqual( fixtures.matchingSuggestions.at );
 		} );
 
-		it( 'should manage the selected suggestion based on both keyboard and mouse events', test( function() {
```

Comment:
Sinon exposes `test` method to enable fake timers. We have fake timers enabled by default in the current Jest configuration.